<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox_OP1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>False</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>True</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_LogOp">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>&amp;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>||</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>!</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>!or</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>==</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_OP2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>False</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>True</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
